source/PlayServicesResolver/src/PlayServicesResolver.cs

```diff
@@ -43,6 +43,11 @@ public class PlayServicesResolver : AssetPostprocessor
         private static IResolver _resolver;
 
         /// <summary>
+        /// Folder to copy the dependencies to
+        /// </summary>
+        private static string _destinationFolder = "Assets/Plugins/Android";
```

Comment:
For this to work, the destination folder would need to at least be "Assets/Plugins/Android".  If you want this to be configurable why not use this as the prefix of the path and allow the user to configure the rest of the path e.g they could set the path to "Aars" which would place aars and exploded aars in "Assets/Plugins/Android/Aars"?